.github/workflows/fulcio-rekor-kind.yaml

```diff
@@ -122,25 +118,84 @@ jobs:
-        done
+        # Make sure the Trillian jobs complete
+        kubectl wait --timeout 5m -n trillian-system --for=condition=Complete jobs --all
+        kubectl delete jobs -n trillian-system --all
 
-        # Now wait for all of the rollouts to complete!
-        for ns in trillian-system rekor-system ctlog-system fulcio-system; do
-          kubectl wait --timeout 10m -n ${ns} --for=condition=Ready pod --all
-        done
+    - name: Install Rekor
+      working-directory: ./src/github.com/sigstore/scaffolding
+      timeout-minutes: 5
+      run: |
+        make ko-apply-rekor
+
+        # Make sure the Rekor jobs complete
+        kubectl wait --timeout 5m -n rekor-system --for=condition=Complete jobs --all
+        kubectl delete jobs -n rekor-system --all
+
+    - if: matrix.k8s-version == 'v1.22.x'
+      working-directory: ./src/github.com/sigstore/scaffolding
+      name: fix the oidc file for 1.22
+      run: |
+        # Backup first
+        cp config/fulcio/fulcio/200-configmap.yaml ./200-configmap.yaml
+        cat config/fulcio/fulcio/200-configmap.yaml| sed 's@https://kubernetes.default.svc.cluster.local@https://kubernetes.default.svc@' > ./200-configmap-new.yaml
+        cp ./200-configmap-new.yaml config/fulcio/fulcio/200-configmap.yaml
+
+    - name: Install Fulcio
+      working-directory: ./src/github.com/sigstore/scaffolding
+      timeout-minutes: 5
+      run: |
+        make ko-apply-fulcio
+
+        # Make sure the Fulcio jobs complete
+        kubectl wait --timeout 5m -n fulcio-system --for=condition=Complete jobs --all
+        kubectl delete jobs -n fulcio-system --all
+
+    - if: matrix.k8s-version == 'v1.22.x'
+      working-directory: ./src/github.com/sigstore/scaffolding
+      name: un-fix the oidc file for 1.22
+      run: |
+        # Restore the backup
+        cp ./200-configmap.yaml config/fulcio/fulcio/200-configmap.yaml
+
+    - name: Install CTLog
+      working-directory: ./src/github.com/sigstore/scaffolding
+      timeout-minutes: 5
+      run: |
+        make ko-apply-ctlog
+
+        # Make sure the CTlog jobs complete
+        kubectl wait --timeout 5m -n ctlog-system --for=condition=Complete jobs --all
+        kubectl delete jobs -n ctlog-system --all
+
+    - name: Setup tuf and start serving it
+      working-directory: ./src/github.com/sigstore/scaffolding
+      timeout-minutes: 5
+      run: |
+        make ko-apply-tuf
+
+        kubectl -n ctlog-system get secrets ctlog-public-key -oyaml | sed 's/namespace: .*/namespace: tuf-system/' | kubectl apply -f -
+        kubectl -n fulcio-system get secrets fulcio-pub-key -oyaml | sed 's/namespace: .*/namespace: tuf-system/' | kubectl apply -f -
+        kubectl -n rekor-system get secrets rekor-pub-key -oyaml | sed 's/namespace: .*/namespace: tuf-system/' | kubectl apply -f -
+
+        # Make sure the tuf jobs complete
+        kubectl wait --timeout 2m -n tuf-system --for=condition=Complete jobs --all
```

Comment:
Suggestion/question: Shouldn't the timeout be 4m ?

Reply:
This one seemed to consistently be quick, but for consistency, I'll change it :)